src/node_file.cc

```diff
@@ -48,6 +48,12 @@
 # include <io.h>
 #endif
 
+#ifndef _WIN32
```

Comment:
I don't mind but it seems like it would be easier to understand if you had...


```
#ifdef _WIN32
#include <windows.h>
#else
#include <unistd.h>
#endif
```

You know?

*If Yagiz is not a man then say "hi gal" else say "hi guy".*

Reply:
lol, true.

Reply:
updated. thanks for the review